nexus/reconfigurator/planning/tests/output/planner_nonprovisionable_bp2.txt

```diff
@@ -0,0 +1,95 @@
+blueprint  9f71f5d3-a272-4382-9154-6ea2e171a6c6
+parent:    55502b1b-e255-438b-a16a-2680a4b5f962
+created by add more Nexus
```

Comment:
In whatever generates this, I think `"add more Nexus"` was probably meant for the `comment` field but got stuffed into the `creator` field.

Reply:
Hmm, looks like this was set in the planner, which doesn't let you set comments externally. I'll change this to be clearer.

Reply:
Changed these.